home.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-0JED4GM2T0"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
+
+    gtag("config", "G-0JED4GM2T0");
+  </script>
+  <title>HTML-CSS-JAVASCRIPT CAPSTONE</title>
+  <!-- Bootstrap CSS -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+
+  <!-- Navigation Menu for Desktop -->
+
+  <nav class="container-fluid d-none d-md-flex flex-column fixed-top desktop-nav">
+    <div class="d-flex justify-content-end top-nav-bar py-1 px-5">
+      <div>
+        <a href="https://www.facebook.com/"><i class="fab fa-facebook-f icon-facebook"></i></a>
+        <a href="https://twitter.com/login"><i class="fab fa-twitter icon-twitter ms-3 me-5"></i></a>
+        <a href="https://www.englishbolo.com/user-login.do" class="text-white top-nav-item">English</a>
+        <a href="#" class="text-white top-nav-item ms-4">MyPage</a>
+        <a href="#" class="text-white top-nav-item ms-4">Logout</a>
+      </div>
+    </div>
+    <div class="d-flex justify-content-between bg-white bottom-nav-bar py-2 px-5">
+      <div class="d-flex flex-row text-wrap justify-content-center logo">
+        <a href="./home.html">Gambit's<br><span class="text-center" id="arena">Arena</span></a>
+      </div>
+      <div class="d-flex justify-content-between align-items-center links-container">
+        <a href="./about.html" class="items fw-bold">About</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Teams</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Players</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Fixtures</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items fw-bold">Tables</a>
+        <a href="https://g.co/kgs/LJBMP1" class="items orange-border py-2 px-3 fw-bold">Latest News</a>
```

Comment:
Hi @Davidosky007,  thanks for the feedback, when I tried #d3d3d3, it had some accessibility checks issue because they were not contrasting, that is the reason why I decided to use the blue color. With this in mind, would you still advise me to change it back to #d3d3d3 or I should stick to the blue color?

Reply:
I have implemented the changes

Reply:
- [ ] This issue is still not resolved. The links are not changing when focused on. See the comments under the review summary. :pray: